book/src/lib/rpc/models.rs

```diff
@@ -6,7 +6,7 @@ pub use helpers::rpc::{Error, RpcResult};
 pub(crate) use crate::db::models::Book as RawBook;
 pub use crate::db::models::*;
 
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
```

Comment:
I was wondering about the `Eq` trait. Why is it necessary here, because of the `Vec` and/or `Option`?

Reply:
My autocompletion added it 😬 
But speaking from the docs it's actually only a marker trait and doesn't add functionality.

Reply:
Okay, because I was a bit confused. The `assert_eq!` does not need it, thus I wanted to ask if you planned something I did not understand/anticipate.